en/faq/nginx-proxy.md

```diff
@@ -35,6 +35,80 @@ server {
 }
 ```
 
+# Using nginx with generated pages and a caching proxy as fallback
+
+If you have a high volume website with regularly changing content, you might want to benefit from Nuxt generate capabilities and [nginx caching](https://www.nginx.com/blog/nginx-caching-guide).
+
+Below is an exampe configuration which you will most likely need to adapt to your needs.
+
+```nginx
+proxy_cache_path  /PATH/TO/YOUR/CACHE levels=1:2 keys_zone=nuxt-cache:25m max_size=1g inactive=60m use_temp_path=off;
+
+map $sent_http_content_type $expires {
+    "text/html"                 1h; # set this to your needs
+    "text/html; charset=utf-8"  1h; # set this to your needs
+    default                     7d; # set this to your needs
+}
+
+server {
+    listen          80;             # the port nginx is listening on
+    server_name     your-domain;    # setup your domain here
+
+    gzip            on;
+    gzip_types      text/plain application/xml text/css application/javascript;
```

Comment:
What about static assets like images?

Reply:
That part I copied from the default nginx proxy example :)

Reply:
But actually its not really useful to gzip images, until you are using tiff's or bmp's images are compressed already so it would be a waste to spent cpu cycles to try to gzip them.

Reply:
You are completely right @pimlie :D